chef/cookbooks/neutron/recipes/l3.rb

```diff
@@ -156,6 +156,16 @@
   )
 end
 
+template "/etc/neutron/dnsmasq-neutron.conf" do
+  source "dnsmasq-neutron.conf.erb"
+  owner "root"
+  group "root"
+  mode "0644"
+  variables(
+    :low_mtu => node[:neutron][:ml2_type_drivers].include?('gre')
```

Comment:
We recently merged support for VXLAN I guess this also requires MTU adjustments.  AFAICS the VXLAN Header has a constant size of 8 bytes.